### PR TITLE
Remove GET for endpoint workspaces/<wid>/time_entries

### PR DESCRIPTION
**Pull request by arturs105**
Backend told that that endpoint exists only for POST/PUT/DELETE (not GET)

### CODE REVIEW

File: Toggl.Ultrawave/ApiClients/TimeEntriesApi.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public IObservable<List<TimeEntry>> GetAll()
         public IObservable<TimeEntry> Create(TimeEntry timeEntry)
         {
             var endPoint = endPoints.Post(timeEntry.WorkspaceId);
-            var observable = CreateObservable<TimeEntry>(endPoint, AuthHeader, timeEntry, SerializationReason.Post);
```
Comment (willsb) [inline Toggl.Ultrawave/ApiClients/TimeEntriesApi.cs]: I don't think this change belongs here

Comment (arturs105) [inline Toggl.Ultrawave/ApiClients/TimeEntriesApi.cs]: Well i had already made changes to that file and XS told me that "Name can be simplified", so i thought i might leave the code a tiny bit cleaner.

Comment (willsb) [inline Toggl.Ultrawave/ApiClients/TimeEntriesApi.cs]: What other changes? Because this is the only change for this file in the Github diff

Comment (arturs105) [inline Toggl.Ultrawave/ApiClients/TimeEntriesApi.cs]: Check the first commit: https://github.com/toggl/mobileapp/pull/203/commits/3bac4647c81244f1d1d844b476fcda82fbc08c67

Comment (arturs105) [inline Toggl.Ultrawave/ApiClients/TimeEntriesApi.cs]: This PR actually was created to add GetForWorkspace method, but then it turned out that the endpoint for that didn't actually exist.

**Pull request merged**